app/main/__init__.py

```diff
@@ -1,8 +1,9 @@
-from flask import Blueprint, current_app
+from flask import Blueprint, current_app, request
```

Comment:
Don't think we need `request` here anymore. The Admin FE has flake8 configured to ignore unused imports in all the `__init__`s, so this is easy to miss!

Reply:
Removed. Well spotted 👍